WBooks/Repositories/BookRepository.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  BookRepository.swift
+//  WBooks
+//
+//  Created by Nacho 2 on 03/04/2020.
+//  Copyright © 2020 Wolox. All rights reserved.
+//
+
+import Foundation
+import Result
+import Alamofire
+
+internal class BookRepository {
```

Comment:
internal is not required

Reply:
its throwing me an error if no declare this class has internal.
I'll recheck the error and send to you

Reply:
![image](https://user-images.githubusercontent.com/24418053/78592128-f863e300-781a-11ea-9a80-17de4ab123c2.png)
In the tutorial guide class is declared as public and then in the final example is internal class.
So if i let public class and method i get this error.
Then following final example i put the class has internal, but i check your comments and removing all (internal of class and public of function) works ok!

Reply:
What if you delete all internal and public in all methods and classes?